content/questions/actions/question-079.md

```diff
@@ -0,0 +1,16 @@
+---
+archetype: "questions"
+title: "Question 079"
+question: "On which commit and branch do scheduled workflows run in GitHub Actions?"
+draft: false
+---
+
+> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+
+1. [ ] Scheduled workflows run on the specific commit on last modified branch.
+   > incorrect, both specific commit and on last modified branch
+1. [ ] Scheduled workflows run on the specific commit on the main branch.
+   > incorrect, both specific commit and main branch
+1. [x] Scheduled workflows run on the latest commit on the default or base branch.
```

Comment:
The term `base` usually refers to the `base` branch of a pull request. I'm quite sure that scheduled workflows run only on default repository branch - if not please correct me
```suggestion
1. [x] Scheduled workflows run on the latest commit on the repository default branch.
```

Reply:
Thank you for your comment! @FidelusAleksander,

![CleanShot 2567-05-21 at 08 27 05@2x](https://github.com/FidelusAleksander/ghcertified/assets/3156572/52ae7f36-d989-44bc-92fe-cb9a5e422b93)

I referenced from the document about the answer `default or base branch`  but as I rechecked it some parts of the document mentioned only the `default branch` as well. 😅

what do you think?

Reply:
I think it's a mistake on their end and scheduled workflows only run on the repository default branch.

Please fix accordingly (remove base) or  if you know how to trigger scheduled runs on other (base) branch please let me know - I would be intrigued and we can merge this question as is🤔

Reply:
Agreed with that. Thank you for your support.